### PR TITLE
Master social app awa

### PR DESCRIPTION
**Pull request by awa-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -43,3 +43,4 @@ XlsxWriter==0.9.3
 xlwt==1.3.*
 xlrd==1.0.0
 pypiwin32 ; sys_platform == 'win32'
+firebase-admin==2.17.0
```
Comment (mreficent) [inline requirements.txt]: @d-fence

Comment (awa-odoo) [reply requirements.txt]: @mreficent It's already being taken care of, thanks

File: addons/web/static/src/js/views/kanban/kanban_view.js [hunk 2]
```diff
@@ -65,6 +65,7 @@ var KanbanView = BasicView.extend({
             archivable: archAttrs.archivable ? !!JSON.parse(archAttrs.archivable) : true,
             group_creatable: activeActions.group_create && !config.device.isMobile,
             quickCreateView: archAttrs.quick_create_view || null,
+            recordsDraggable: archAttrs.records_draggable ? !!JSON.parse(archAttrs.records_draggable) : true,
```
Comment (VincentSchippefilt) [inline addons/web/static/src/js/views/kanban/kanban_view.js]: missing doc in views rst & a test

File: addons/website/static/lib/jstz.min.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+/* jstz.min.js Version: 1.0.6 Build date: 2015-11-04 */
```
Comment (VincentSchippefilt) [inline addons/website/static/lib/jstz.min.js]: you should add the non minified version of the lib and let the bundle of odoo do the minification (like all other libs), also make sure to commit the license if any

File: addons/website/static/src/js/visitor_timezone.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+var utils = require('web.utils');
+var publicWidget = require('web.public.widget');
+
+publicWidget.registry.visitorTimezone = publicWidget.Widget.extend({
+    selector: '#wrapwrap',
+
+    start: function () {
+        if (!localStorage.getItem('website.found_visitor_timezone')) {
+            var timezone = jstz.determine().name();
+            this._rpc({
```
Comment (VincentSchippefilt) [inline addons/website/static/src/js/visitor_timezone.js]: no return on purpose? 
no call to super on purpose ?

File: addons/website/models/website_visitor.py [hunk 5]
```diff
@@ -140,6 +142,10 @@ def _prepare_visitor_send_mail_values(self):
             }
         return {}
 
+    @api.model
+    def _tz_get(self):
```
Comment (tde-banana-odoo) [inline addons/website/models/website_visitor.py]: from odoo.addons.base.models.res_partner import _tz_get

**Comment by tde-banana-odoo** [comment]
@VincentSchippefilt @dbeguin only moved code from enterprise (about library). We can check for the non minimized version and add requested tests in an incoming task (FP priorities = vendredi merge).

Bisous tout plein.

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**